Makefile

```diff
@@ -1,23 +1,29 @@
 .PHONY: check
 check: lint cs tests phpstan
 
+vendor:
```

Comment:
Isn't Makefile's purpose to skip targets if the directory with the same name exists? Isn't this going to cause `composer install` to be skipped if `vendor` exists?

Reply:
Yeah it is, i'll make it depend on composer.json/composer.lock to make it update if one of those does